src/main.py

```diff
@@ -15,6 +16,7 @@
 app.logger.info('.env loaded')
 
 # TODO: ADD RESOURCES HERE
+api.add_resource(SingleReview, "/review/<string:_id", endpoint = "drink")
```

Comment:
A couple of things here:

- the URL should be `/reviews` (plural)
- route parameter in the URL missing `>`
- endpoint is `"review"`.